relational_operators/CreateIndexOperator.hpp

```diff
@@ -85,7 +78,7 @@ class CreateIndexOperator : public RelationalOperator {
  private:
   CatalogRelation *relation_;
   const std::string &index_name_;
-  const std::vector<IndexSubBlockDescription> index_descriptions_;
+  IndexSubBlockDescription index_description_;
```

Comment:
@zuyu I removed const from this member instance because I am moving this in updateCatalogOnCompletion() method. Should I then favour not moving this and pass it by reference?